service/controller/v18/key/key.go

```diff
@@ -29,7 +29,7 @@ const (
 	// ProbeHost host for liveness probe.
 	ProbeHost = "127.0.0.1"
 	// InitialDelaySeconds is InitialDelaySeconds param in liveness probe config
-	InitialDelaySeconds = 240
+	InitialDelaySeconds = 360
```

Comment:
Why is that 6 minutes now?

Reply:
Are there opportunities to speed this up in the future? SIG operator would like to pursue a main objective of fast transitioning times, which includes tenant cluster creation.

Reply:
I've spent a lot of time here and the only way we can speed up the process - burning all static configurations into  coreos image. That would require CI process for building pxe images, ami images for amazon and azure images.